src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -62,16 +70,20 @@ type State = {|
 class TaskForm extends React.Component<Props, State> {
     static defaultProps = {
         approverSelectorContacts: [],
+        dueDate: null,
+        editMode: TASK_EDIT_MODE_CREATE,
+        message: '',
```

Comment:
this is marked as required since one cannot have a task without a message. the case here is when we create a new task, we still want a required field, but it starts empty.